core/src/components/modal/test/card/modal-tablet.e2e.ts

```diff
@@ -61,12 +61,12 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       test('it should not swipe to close when swiped on the content but the content is scrolled', async ({ page }) => {
         const modal = await cardModalPage.openModalByTrigger('#card');
 
-        const content = (await page.$('ion-modal ion-content'))!;
+        const content = page.locator('ion-modal ion-content');
         await content.evaluate((el: HTMLIonContentElement) => el.scrollToBottom(0));
 
         await cardModalPage.swipeToCloseModal('ion-modal ion-content', false);
 
-        await content.waitForElementState('stable');
+        await content.waitFor();
```

Comment:
Why was this change made?

Reply:
Playwright [discourages](https://playwright.dev/docs/api/class-elementhandle) the use of `ElementHandle`. `waitForElementState` is based on that class. It recommends to use `Locators` instead.

Reply:
Ok makes sense 👍